pages/services.vue

```diff
@@ -1,11 +1,61 @@
 <template>
-  <iframe
-    class="airtable-embed h-full min-h-screen"
-    src="https://airtable.com/embed/shra12KjCQJwrnu8l?backgroundColor=gray&layout=card&viewControls=on"
-    frameborder="0"
-    onmousewheel=""
-    width="100%"
-    height="100%"
-    style="background: transparent; border: 0px solid #ccc"
-  ></iframe>
+  <div class="bg-transparent shadow overflow-hidden sm:rounded-md h-full min-h-screen">
+    <ul role="list" class="divide-y divide-gray-200" v-for="service in services" :key="service.id">
+      <TServiceListItem
+        :fieldId="service.fields.id"
+        :name="service.fields.Name"
+        :comments="service.fields.Comments"
+        :url="service.fields.URL"
+        :location="service.fields.Location"
+        :type="service.fields.Type"
+      />
+    </ul>
+  </div>
 </template>
+<script>
+import TServiceListItem from '../components/TServiceListItem.vue'
+export default {
+  name: 'services',
+  data: () => ({
+    services: [],
+  }),
+  async created() {
+    await this.getServices()
+    //console.log(this.services[29]);
+  },
+  components: {
+    TServiceListItem,
+  },
+  methods: {
+    async getServices() {
+      const option = {
+        headers: { 
+            'Authorization': `Bearer ${this.$config.API_KEY}`, 
+        }
+      }
+      let offset = null
+      try {
+         do {
+          const off_set = offset ? `offset=${offset}` : ''
+          const url = `https://api.airtable.com/v0/${this.$config.BASE_ID}/Services?view=Grid%20view&${off_set}`
+          const response = await $fetch(url, option)
+          if(response?.offset){
+            this.setServices(response.records)
+            offset = response.offset
+          }else {
+            this.setServices(response.records)
+            offset = null
+          }
+        } while (offset)
+      } catch (error) {
+        throw error
+      }
+    },
+    setServices(services) {
+      services.forEach(element => {
+          this.services.push(element)
+      });
+    },
+  },
+}
+</script>
```

Comment:
As a user, I push items on a list and expect the URL to be opened. Is that in scope or out of scope?

Reply:
not entirely sure. I made this based on the live version of the web, and it worked similarly

Reply:
Then I think need to notify @Kateryna Vasylenko issue could be added to the backlog for next week if it is important, if not discarded or saved for later

Reply:
out of scope

Reply:
out of scope